spring-cloud-contract-stub-runner/src/main/java/org/springframework/cloud/contract/stubrunner/GitRepo.java

```diff
@@ -290,13 +290,13 @@ private boolean containsBranch(Git git, String label,
 		return false;
 	}
 
-	private void deleteBaseDirIfExists() {
+	private void deleteBaseDirIfExists(String errorMessage) {
 		if (this.basedir.exists()) {
 			try {
-				FileUtils.delete(this.basedir, FileUtils.RECURSIVE);
+				org.apache.commons.io.FileUtils.deleteDirectory(this.basedir);
```

Comment:
Changed the apache commons as in the jgit lib it does not like deleting the pack.idx file from .git 
Apachecommons deletes correctly as it is handling the windows file path issues.